app/lib/submission_builder/return_header.rb

```diff
@@ -12,6 +12,10 @@ def document
           xml.TaxPeriodEndDt date_type(Date.new(@submission.data_source.tax_return_year, 12, 31))
         end
         xml.TaxYr @submission.data_source.tax_return_year
+        xml.PaidPreparerInformationGrp do
```

Comment:
Lmk if we should gate this behind a `if @submission.data_source.state_code_nj?` or similar

Reply:
it seems safer to gate it, if there's a precedent/model for doing so -- how difficult would that be?

Reply:
Just added conditional logic! Will pull in a reviewer from CfA to see if this approach is okay.

Reply:
We strongly prefer not to use state codes in conditional branches if it can be avoided. Another different approach could be to:
* define methods for `ptin` and `preparer_person_name` in `nj_return_xml.rb` that return these hardcoded values
* define a default implementation in `document.rb` that returns nil
* check for the existence of the fields before adding them

If you need to rush this for testing though, this can be a follow up ticket

Reply:
Appreciate the guidance on an alternate approach. Agree that that'd be less brittle than relying on `state_code`. After talking with our PM, we'd like to merge this for vendor testing next week.

We'll refactor with this ticket, prioritized for next week: https://github.com/newjersey/affordability-pm/issues/230#issue-2753443632